hazelcast/src/main/java/com/hazelcast/internal/yaml/YamlLoader.java

```diff
@@ -50,6 +50,7 @@ public static YamlNode load(InputStream inputStream, String rootName) {
         try {
             YamlDocumentLoader load = getLoad();
             Object document = load.loadFromInputStream(inputStream);
+            inputStream.close();
```

Comment:
Similar comment as Matko's one here : https://github.com/hazelcast/hazelcast/pull/15029#discussion_r284322486. It's better that the creater of inputStream will close it as well.

Reply:
:+1: Backported the change.